gatsby-node.js

```diff
@@ -485,26 +489,47 @@ const processOpenApiFiles = async (
     if (data.allFile.edges.length > 0) {
       await asyncForEach(data.allFile.edges, async ({ node }) => {
         let filepath = node.absolutePath
+        let ext = node.ext
+        let content = node.internal.content
         let seo = searchTree(
           parliamentNavigation.pages,
           `${node.name}${node.ext}`
         )
-
         try {
-          const swaggerObject = await SwaggerParser.bundle(filepath)
-          if (Object.keys(swaggerObject.paths).includes("")) {
-            swaggerObject.paths = renameProp("", "/", swaggerObject.paths)
+          let openapi = false
+          if (ext === '.json') {
+            let json = JSON.parse(content)
+            if(json.openapi) {
+              openapi = true
+            }
+          } else {
+            let yaml = YAML.parse(content)
+            if(yaml.openapi) {
```

Comment:
Need to add a check in here for Swagger files. Remember the `openapi` property was only introduced in 3.0 for 2.0 files the check would be `if (json.swagger)` for example.

Reply:
Good catch! I updated the code to check for `json.swagger` and `yaml.swagger` as well.